### PR TITLE
Add help message related to some apps options.

### PR DESCRIPTION
**Pull request by FdaSilvaYY**
As there is some common options between apps, some helps message can be replicated.

### CODE REVIEW

File: apps/cms.c [hunk 1]
```diff
@@ -177,7 +177,7 @@ OPTIONS cms_options[] = {
     {"noindef", OPT_NOINDEF, '-'},
     {"nooldmime", OPT_NOOLDMIME, '-'},
     {"crlfeol", OPT_CRLFEOL, '-'},
-    {"noout", OPT_NOOUT, '-'},
+    {"noout", OPT_NOOUT, '-', "Don't produce any output"},
```
Comment (richsalz) [inline apps/cms.c]: Change to "Do not"; let's avoid contractions. :)

**Comment by richsalz** [comment]
Thanks for doing this.  One minor edit and then we can pull it in.

**Comment by FdaSilvaYY** [comment]
Edit done with a few first-letter case fix.
Ready for pull, IMHO.

**Comment by richsalz** [comment]
+1

File: apps/ca.c [hunk 2]
```diff
@@ -238,7 +238,7 @@ OPTIONS ca_options[] = {
      "msie modifications to handle all those universal strings"},
     {"crldays", OPT_CRLDAYS, 'p', "Days is when the next CRL is due"},
     {"crlhours", OPT_CRLHOURS, 'p', "Hours is when the next CRL is due"},
-    {"crlsec", OPT_CRLSEC, 'p'},
+    {"crlsec", OPT_CRLSEC, 'p', "Seconds is when the next CRL is due"},
```
Comment (mattcaswell) [inline apps/ca.c]: Seconds *until" the next CRL is due? Also probably should be the same for crldays and crlhours.

File: apps/cms.c [hunk 3]
```diff
@@ -142,7 +142,7 @@ OPTIONS cms_options[] = {
     {"encrypt", OPT_ENCRYPT, '-', "Encrypt message"},
     {"decrypt", OPT_DECRYPT, '-', "Decrypt encrypted message"},
     {"sign", OPT_SIGN, '-', "Sign message"},
-    {"sign_receipt", OPT_SIGN_RECEIPT, '-'},
+    {"sign_receipt", OPT_SIGN_RECEIPT, '-', "Sign CMS message receipt structure"},
```
Comment (mattcaswell) [inline apps/cms.c]: Better would be "Generate a signed receipt for the message"

File: apps/cms.c [hunk 4]
```diff
@@ -177,7 +177,7 @@ OPTIONS cms_options[] = {
     {"noindef", OPT_NOINDEF, '-'},
     {"nooldmime", OPT_NOOLDMIME, '-'},
     {"crlfeol", OPT_CRLFEOL, '-'},
-    {"noout", OPT_NOOUT, '-'},
+    {"noout", OPT_NOOUT, '-', "Do not produce any output"},
```
Comment (mattcaswell) [inline apps/cms.c]: "For the -cmsout operation do not output the parsed CMS structure"

File: apps/cms.c [hunk 5]
```diff
@@ -191,7 +191,7 @@ OPTIONS cms_options[] = {
      "Do not load certificates from the default certificates directory"},
     {"content", OPT_CONTENT, '<',
      "Supply or override content for detached signature"},
-    {"print", OPT_PRINT, '-'},
+    {"print", OPT_PRINT, '-', "Print CMS ContentInfo details"},
```
Comment (mattcaswell) [inline apps/cms.c]: "For the -cmsout operation print out all fields of the CMS structure"

File: apps/cms.c [hunk 6]
```diff
@@ -205,7 +205,7 @@ OPTIONS cms_options[] = {
     {"signer", OPT_SIGNER, 's', "Signer certificate file"},
     {"recip", OPT_RECIP, '<', "Recipient cert file for decryption"},
     {"certsout", OPT_CERTSOUT, '>', "Certificate output file"},
-    {"md", OPT_MD, 's'},
+    {"md", OPT_MD, 's', "Any supported digest"},
```
Comment (mattcaswell) [inline apps/cms.c]: Digest algorithm to use when signing or resigning.

File: apps/smime.c [hunk 7]
```diff
@@ -148,7 +148,7 @@ OPTIONS smime_options[] = {
     {"rand", OPT_RAND, 's',
      "Load the file(s) into the random number generator"},
     {"passin", OPT_PASSIN, 's', "Input file pass phrase source"},
-    {"md", OPT_MD, 's'},
+    {"md", OPT_MD, 's', "Any supported digest"},
```
Comment (mattcaswell) [inline apps/smime.c]: "Digest algorithm to use when signing or resigning"

**Comment by FdaSilvaYY** [comment]
Messages fixed, and code rebased onto current master.

**Comment by mattcaswell** [comment]
+1

ping @richsalz - a few things have changed since your review. You may want to verify (and push if you want to).

**Comment by richsalz** [comment]
+1 I'll merge.
(BTW, asn1parse help text doesn't start with an uppercase letter; I wonder if others are broke?)

**Comment by richsalz** [comment]
commit 16e1b28 pushed, thanks.

**Pull request closed without merging**